detection-rules/impersonation_norton_lifelock.yml

```diff
@@ -1,12 +1,16 @@
-name: "Brand impersonation: Norton Lifelock"
+name: "Brand impersonation: Norton"
 description: |
-  Scans files to detect Norton Lifelock impersonation.
+  Scans files to detect Norton (Lifelock) impersonation.
 references:
   - "https://techaeris.com/2021/10/23/psa-watch-out-for-norton-lifelock-phishing-emails/"
+  - "https://playground.sublimesecurity.com?id=31310ef5-8725-49b8-9c33-6b18ecdb5ba0"
+  - "https://playground.sublimesecurity.com?id=a67bc61c-28f0-4904-a046-4584e706697d"
+  - "https://playground.sublimesecurity.com?id=3f5809c9-43cf-4f0c-a709-6f9e3912dbb4"
 type: "rule"
 severity: "low"
 source: |
   type.inbound
+  and sender.email.domain.domain != "norton.com"
   and any(attachments,
       .file_extension in~ ('pdf', 'jpg', 'jpeg', 'png')
```

Comment:
should we make it SOP to also check `.file_type` any time we're checking `.file_extension`?